articles/Monerium/monerium.md

```diff
@@ -0,0 +1,155 @@
+Asset Risk Assessment - Monerium
+===============
+### A risk assessment of e-money, the Monerium (EURe) stablecoin, and its risks for Curve LPs
+ 
+This research was spearheaded by [@evmknows](https://twitter.com/evmknows).
+ 
+## Useful links
+ 
+ [Website](https://monerium.com/)  
+ [Documentation](https://monerium.dev/)  
+ [Github](https://github.com/monerium/)  
+ [Gauge request](https://gov.curve.fi/t/proposal-to-add-ageur-eure-on-ethereum-to-the-gauge-controller/4544)
+ 
+## Abstract
+
+In this asset risk assessment, we will cover the EURe stablecoin issued by [Monerium](https://monerium.com/) prompted by [the request to deploy an agEUR/EURe gauge](https://gov.curve.fi/t/proposal-to-add-ageur-eure-on-ethereum-to-the-gauge-controller/4544). Since the stablecoin is issued under the strict regulatory requirements imposed by the [e-money directive](https://eur-lex.europa.eu/legal-content/en/TXT/?uri=CELEX%3A32009L0110) we decided to elaborate on the concept of e-money and its regulations. If you are familiar with the latter and only interested in the Monerium review you can skip to the part by clicking here.
+
+A quick TL;DR of our findings:
+
+* Monerium offers a EURe stablecoin which can be on- and off-boarded using an [IBAN](https://en.wikipedia.org/wiki/International_Bank_Account_Number) (International Bank Account Number). When registering  with Monerium you get a personal IBAN that you can fund in order to receive EURe on your on-chain wallet. In the same way, you can off-board EURe by funding your on-chain wallet and sending the EURe via the Monerium app to an IBAN. By doing that the EURe is directly burnt from your wallet.
+* Monerium is regulated as an electronic money institution (EMI) for the issuance of e-money.
+* An EMI must meet a number of regulatory requirements, such as over-collateralization with €350k and a 2% buffer based on the outstanding issued amount of e-money, segregating the EMIs funds and reserves by using different custodians and investing the reserves into secure, low-risk assets denominated in euro.
```

Comment:
I'd say over-collateralization is the fitting term because (IIRC) the 350k and 2% is a buffer to cover residual risks related to the safeguarding requirements (can't find the remark rn). Even in the event that for example the HQLAs are sold for a slight loss, and the EMI filed for bankruptcy, the emoney holders have a priority claim.

https://monerium.com/policies/business-terms-of-service/

> According to the Icelandic Bankruptcy Act No. 21/1991, e-money holders enjoy the highest priority claim against the bankruptcy estate in proportion to their ownership. In the event of our insolvency, or if a financial claim is made against us, no creditor or claimant should be able to claim funds held in our Safeguarded Accounts. This is because no other person or institution may have any rights or interest over the funds held in our Safeguarded Accounts such as a lien.

But even without concrete statements from the EMD & MiCA that the 350k and 2% constitutes a buffer to cover residual risks. Because of Article 44(4)

> Upon request by the holder of e-money tokens, the respective issuer must redeem, at any moment and at par value the monetary value of the e-money tokens held to the holders of e-money tokens, either in cash or by credit transfer.

It can be assumed that own capital will be used to fulfill the obligations otherwise the EMI risks the loss of his license, fines and potentially jail time for the people in charge?

The above interpretation is supported by the fact that significant e-money tokens have and additional requirement where it needs 3% instead of 2%

> The percentage referred to in Article 31(1), point (b), shall be set at 3% of the average amount of the reserve assets for issuers of significant asset-referenced tokens.
(^^ applies to Sig. EMT as per Article 52 of MiCA)

The profitability of a stablecoin issuer likely scales faster than the outstanding issued e-money tokens. Thus, the extra 1% only makes sense as an additional buffer to secure the reserves.